config/config.py

```diff
@@ -12,8 +12,8 @@
 }
 
 logging = {
-    'root': {'level': 'INFO', 'handlers': ['console']},
     'loggers': {
+        'root': {'level': 'INFO', 'handlers': ['console']},
```

Comment:
this is actually wrong (from my own achievement nonetheless!) the right approach is how you had it. See: https://github.com/pecan/pecan/pull/40